Anagrams/src/ua/goit/alg/Anagrams.java

```diff
@@ -0,0 +1,28 @@
+package ua.goit.alg;
+import java.util.Scanner;
+
+public class Anagrams {
+
+    public static void main(String[] args) {
+	Scanner sc = new Scanner(System.in);
+	String input = sc.nextLine(); 
+	sc.close();
+	doAnagram(input);
+    }
+
+    public static String doAnagram(String input) {
```

Comment:
Anagrams task mandates no additional memory usage, so no 'split', stringbuilder, etc. Just put the result into the System.out